tests/integration/commands/dev/edge-functions.test.ts

```diff
@@ -30,9 +30,9 @@ const routes = [
   },
 ]
 
-describe('edge functions', () => {
+describe.skipIf(isWindows)('edge functions', () => {
```

Comment:
why are they skipped on windows? shouldn't it work there as well?

Reply:
https://github.com/netlify/cli/blob/e08a47f2d362fa82a64237260b5bc517ca75a6b3/tests/integration/commands/dev/edge-functions.test.ts#L9-L10